pkg/steps/clusterinstall/template.go

```diff
@@ -322,13 +322,21 @@ objects:
         }
 
         function run-upgrade-tests() {
-          REPOSITORY_PATH=/resourcewatch openshift-tests run-resourcewatch --kubeconfig=${KUBECONFIG} --namespace=ci-operator-monitor &
-          rw_pid=$!
+          openshift-tests run-resourcewatch --help > /dev/null 2>&1
+          rwsupported=$?
```

Comment:
I think these two lines need to be changed. Could you replace the 325-326 with these two lines?
 ```
          rwsupported=0   
          openshift-tests run-resourcewatch --help > /dev/null 2>&1 || rwsupported=$?
```

Reply:
well, this is a pretty bad look. We're hosed again.

I'll try one last tiem to fix it

Reply:
We use 'set -e' that is really the trouble maker. I don't like `set -e` at all. Since it fails with any error, we have to invent these kind of workaround that makes the scripts look very ugly. 

I have written lengthy shell scripts and maintained them, while taking care of my own error handling. Never had a problem with scripts or error handling.